### PR TITLE
Closes #23 Added Contact Us Page

### LINKED ISSUE #23: [OTHER] Add Contact Us Page
### What would you like to share or ask?

In navbar there is a field for contact us but it does not redirect to any page.Hence I would like to create a contact us page wherein the contact details will be provided and a form can be included using formspree for contacting via any message 

### Additional information

open zone
Here the Contact Us option is seen but a page for the same isn't available, hence I am raising this issue. Please assign me this under GSSoC'24.
Thank you!

### PR DESCRIPTION
**Pull request by komal-agarwal5**
## Fixes Issue
Issue #23 has been fixed.

## Changes proposed
I have added a contact us page which gets redirected when clicked either from navbar or footer.There form layout is there to type in suggestions or any queries.

## Screenshots
contactus

## Note to reviewers
NA

### CODE REVIEW

File: package.json
Comment (AbhiPatel10) [inline package.json:1]: Please remove this file Changes

File: pnpm-lock.yaml
Comment (AbhiPatel10) [inline pnpm-lock.yaml:1]: Please remove this file Changes

Comment (komal-agarwal5) [reply pnpm-lock.yaml:1]: ok working on it

**Comment by komal-agarwal5** [comment]
@AbhiPatel10 I have made the asked changes,will you please see once?

**Review by AbhiPatel10 — APPROVED** [review]
Looks good to me

@komal-agarwal5 , Thanks for your contribution.

**Pull request merged**